external/README.md

```diff
@@ -64,12 +64,12 @@
 - Version: 1.12.0
 - License: Apache-2.0
 
-## freetype
+## Freetype
```

Comment:
shoulde be FreeType?

Reply:
Please add the whole request. I will look later for correct wording.
Thanks.